Makefile

```diff
@@ -2,7 +2,6 @@ MAKEFILE_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
 
 .PHONY: bootstrap
 bootstrap:
-	./scripts/install-toolchain.sh
```

Comment:
Can this script file be deleted too?

Reply:
It's still useful for installing the default toolchain without carton, and I still use it.